common/src/main/java/com/movie/recommender/common/model/location/LocationDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.movie.recommender.common.model.location;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class LocationDTO {
```

Comment:
move `LocationDTO` to the `common` project and update `LocationService` to use this model from the common project
it seems you copied the `Location` entity instead of the DTO